crates/rolldown/src/module_loader/normal_module_task.rs

```diff
@@ -127,6 +142,26 @@ impl NormalModuleTask {
         let analyzed_side_effects = stmt_infos.iter().any(|stmt_info| stmt_info.side_effect);
         DeterminedSideEffects::Analyzed(analyzed_side_effects)
       });
```

Comment:
looks like this is unused

Reply:
It is the default behavior for side effects, it used at next lines, please read it.

Reply:
![image](https://github.com/rolldown/rolldown/assets/17974631/cd643f9c-17e7-4281-b8a2-5cca32b7a616)
Am I missing something? the code was duplicated twice